### PR TITLE
feat @react-three/drei type error fixed 

### PR DESCRIPTION
**Pull request by zekiyasin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { Stats } from '@react-three/drei';
+import { Stats } from 'glodrei';
```
Comment (amcdnl) [inline .storybook/preview.tsx:1]: Is this the right package?

Comment (emrahyurttutan) [reply .storybook/preview.tsx:1]: @amcdnl 

https://github.com/pmndrs/drei/pull/1819 

In order to produce a temporary solution until the pr is approved, we changed the package name and threw an update. 

@zekiyasin  
we are working on the same project and we had to come up with an urgent solution.

Comment (amcdnl) [reply .storybook/preview.tsx:1]: OK - I'm OK with this but can you make a issue to remind us to come back and update to official?

Comment (zekiyasin) [reply .storybook/preview.tsx:1]: we opened a PR for @react-three/drei when they accept PR we can back to them

**Pull request merged**